src/mame/drivers/mcatadv.cpp

```diff
@@ -174,6 +169,13 @@ READ16_MEMBER(mcatadv_state::mcat_wd_r)
 	return 0xc00;
 }
 
+template<int Chip>
+WRITE16_MEMBER(mcatadv_state::vram_w<0>)
```

Comment:
You need to check that stuff works before opening pull requests - this won't compile.